src/DependencyInjection/Configuration.php

```diff
@@ -20,23 +21,12 @@ public function getConfigTreeBuilder()
         /** @var ArrayNodeDefinition $root */
         $root = $treeBuilder->getRootNode();
 
+        $this->addAuth0SdkConfiguration($root);
+
         $root
             ->children()
-                ->scalarNode('login_domain')->defaultNull()->info('If you configured SSO with a custom domain.')->end()
```

Comment:
I think we need to keep login_domain. That is used for the `Auth0EntryPoint`. If you have configured a custom domain for the login, you will use one domain for login and one for the API.

Reply:
done